wiki/Community/Project_Loved/en.md

```diff
@@ -23,32 +23,25 @@ Creators of said beatmaps are also able to request them to be removed from the L
 
 There are some very minimal criteria that need to be met for beatmaps to be nominated for Loved. Each game mode has its own specific criteria for Loved beatmap nomination and priority, but all modes follow these rules:
 
-- It is not in the Qualified, Ranked, or Approved categories
+- It abides by the [beatmap submission rules](/wiki/Rules#beatmap-submission-rules)
+- It is not in the [Ranked](/wiki/Beatmap/Category#ranked), [Approved](/wiki/Beatmap/Category#approved), or [Qualified](/wiki/Beatmap/Category#qualified) categories
 - Its creator does not disapprove of it being Loved
-- Its creator is not restricted
+  - If any guest contributors disapprove of their part being Loved, their part will be excluded from Loved by appropriate means. In some cases, such as [collabs](/wiki/Beatmap/Beatmap_collaborations), this may make the entire beatmap ineligible for Loved.
+- Its creator is not [restricted](/wiki/Help_centre/Account_restrictions)
 - At least one difficulty can be passed
+- Every nominated difficulty has a [drain time](/wiki/Beatmap/Drain_time) of at least 20 seconds
 
-Although many Pending and Work-in-progress beatmaps meet these criteria, few are voted into the Loved category due to the more selective nomination and voting requirements.
+Although many [Work in Progress and Pending](/wiki/Beatmap/Category#wip-and-pending) beatmaps meet these criteria, few are voted into the Loved category due to the selective nature of the captains' picks, and the high community approval requirement by voting.
 
-Beatmaps entering the Loved category are required to follow some additional rulesets:
-
-- [Some points from the Metadata section of the ranking criteria](/wiki/Ranking_criteria#metadata):
-  - Song title, artist, and source should be correct in all cases.
-  - Beatmap tags should properly list all contributors (guest difficulty creators, storyboarders, etc.).
-- [Song Content Rules](/wiki/Rules/Song_content_rules)
-- [Visual Content Considerations](/wiki/Rules/Visual_content_considerations)
-
-Additionally, in order to keep the audio quality of the song at a reasonable level, beatmap audio files should have an average bitrate of at least 128 kbps.
-
-Any problems with these rules are usually sorted out while the beatmap is up for voting.
+Contributors to Project Loved often suggest technical changes to maps before they are Loved, such as improvements to audio quality, [timing](/wiki/Beatmapping/Timing) accuracy, or [metadata standardisation as suggested by the ranking criteria](/wiki/Ranking_criteria#metadata). The mapper is strongly encouraged to apply these changes, but they are not required to.
```

Comment:
should add back in the song content rules and visual content considerations, these are not part of the metadata check but instead the moderation check and are required changes before the map can be loved, unless the osu! team decides otherwise.

Reply:
those are part of submission rules

Reply:
Nvm changed this part to clarify